### PR TITLE
Ei 364 get employer vendor

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Api/Controllers/AccountCommandController.cs [hunk 1]
```diff
@@ -45,5 +45,14 @@ public async Task<IActionResult> UpdateVendorRegistrationCaseStatus([FromRoute]
             await SendCommandAsync(new UpdateVendorRegistrationCaseStatusCommand(hashedLegalEntityId, request.CaseId, request.Status, request.CaseStatusLastUpdatedDate));
             return NoContent();
         }
+
+        [HttpPut("/legalentities/{hashedLegalEntityId}/employervendorid")]
+        public async Task<IActionResult> AddEmplyerVendorId([FromRoute] string hashedLegalEntityId, [FromBody] AddEmployerVendorIdRequest request)
```
Comment (David-B-Read) [inline src/SFA.DAS.EmployerIncentives.Api/Controllers/AccountCommandController.cs:50]: Typo in action name

File: src/SFA.DAS.EmployerIncentives.Commands/AddEmployerVendorIdForLegalEntity/AddEmployerVendorIdForLegalEntityCommandValidator.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                result.AddError("HashedLegalEntityId", "Is not set");
+            }
+
+            if (item.EmployerVendorId == default)
+            {
+                result.AddError("EmployerVendorId", "Is not set");
+            }
+            else if (item.EmployerVendorId.All(c => c == '0'))
+            {
+                result.AddError("EmployerVendorId", "Is not set to a validate Id");
```
Comment (David-B-Read) [inline src/SFA.DAS.EmployerIncentives.Commands/AddEmployerVendorIdForLegalEntity/AddEmployerVendorIdForLegalEntityCommandValidator.cs:25]: Should that error message be "Is not set to a valid Id" ?

Comment (Pauljgraham) [reply src/SFA.DAS.EmployerIncentives.Commands/AddEmployerVendorIdForLegalEntity/AddEmployerVendorIdForLegalEntityCommandValidator.cs:25]: fixed

File: src/SFA.DAS.EmployerIncentives.Api/Controllers/AccountCommandController.cs (hunk 1 above)
Comment (Pauljgraham) [reply src/SFA.DAS.EmployerIncentives.Api/Controllers/AccountCommandController.cs:50]: thx

**Review by David-B-Read — APPROVED** [review]

**Review by Stephek — APPROVED** [review]

**Pull request merged**